app/views/projects/show.html.erb

```diff
@@ -34,7 +34,7 @@
       <p class="fst-italic">There hasn't been any recent workspace activity in this project.</p>
     <% end %>
 
-    <% favorites = @project.workspaces.favorites.accessible_by(current_ability) %>
+    <% favorites = current_user.favorited_workspaces %>
```

Comment:
Still check accessibility here or?

Reply:
Yes, I think so? it's possible to favorite a public workspace, and then the workspace owner might take it private again?